src/app/sketch/sketch.ts

```diff
@@ -29,8 +29,12 @@ export class Sketch {
 
   constructor(sketch: SketchInterface) {
     this.id = sketch.id;
-    this.boards = sketch.boards;
-    this.links = sketch.links;
+    this.boards =  Array.from( sketch.boards, ( b: BoardInterface ) => {
```

Comment:
daca zici intr-un loc e usor s zici doar sa fac asta si in celelalte parti :)), nu tre sa pui la toate =))